docs/configuration.asciidoc

```diff
@@ -148,6 +148,27 @@ NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]
 | Name of the entry assembly       | String 
 |============
 
+[float]
+[[config-service-node-name]]
+==== `ServiceNodeName` (added[1.2])
+
+If set, this name is used to distinguish between different nodes of a service.
+If not set, data aggregations will be done based on a container ID (where valid) or on the reported hostname (automatically discovered).
+
+NOTE: Metrics views can utilize this configuration since APM Server 7.5
+
+[options="header"]
+|============
+| Environment variable name       | IConfiguration or Web.config key 
+| `ELASTIC_APM_SERVICE_NODE_NAME` | None
```

Comment:
The `IConfiguration or Web.config key ` part is TBD, if we all agree we want to have that, this needs to be updated. If not, then sorry for the noise :)

Reply:
done